src/main/external-resources/UMS.conf

```diff
@@ -803,26 +803,6 @@ use_embedded_subtitles_style =
 # Default: "0"
 3d_subtitles_depth =
 
-# Live subtitles limit
-# --------------------
-# The maximum number of live subtitles results to show.
-# Default: 20
-live_subtitles_limit =
-
-# Keep live subtitles
-# -------------------
-# Whether to keep downloaded live subtitles permanently.
-# See live_subtitles_timeout below.
-# Default: false
-live_subtitles_keep =
```

Comment:
@SubJunk Of course we do ;-)

These settings comments are already write there:
https://github.com/UniversalMediaServer/UniversalMediaServer/blob/03448e7321761fcc67eaf350fa5ab230162af9eb/src/main/external-resources/UMS.conf#L1414-L1436